pyuavcan/transport/can/media/pythoncan/_pythoncan.py

```diff
@@ -16,42 +24,174 @@
 class PythonCANMedia(_media.Media):
     """
     A media interface adapter for `python-can <https://github.com/hardbyte/python-can>`_.
-    This is a stub, the implementation is missing. Please submit patches!
+      * Usage example for PCAN-USB channel 1 (bitrate = 500k, mtu = 8, Node-ID = 10): 
+          --tr="CAN(can.media.pythoncan.PythonCANMedia('pcan','PCAN_USBBUS1',5000000,8),10)"
+      * Usage example for Kvaser channel 0 (bitrate = 500k, mtu = 8, Node-ID = 10): 
+          --tr="CAN(can.media.pythoncan.PythonCANMedia('kvaser','0',5000000,8),10)"
     """
 
-    def __init__(self) -> None:
-        raise NotImplementedError
+    def __init__(self, iface_name: str, channel_name: str, bitrate: int, mtu: int) -> None:
```

Comment:
At the moment second bitrate is the same, but we can change it, python-can supports some FD features, so it should be possible to add data bitrate there.

I don't think it's a good idea to limit CAN FD packet with 64 bytes only, we have to add all possible packet sizes here.

May be could concatenate interface name and channel name as a ':'-separated string and process it internally?

Reply:
> I don't think it's a good idea to limit CAN FD packet with 64 bytes only, we have to add all possible packet sizes here.

I am not saying we should do that. I am saying that most applications will be using 64 bytes, so it makes sense to make it a default, so that the user won't have to type the `,8` at the end in `can.media.pythoncan.PythonCANMedia('pcan','PCAN_USBBUS1',5000000,8)`.

> May be could concatenate interface name and channel name as a ':'-separated string and process it internally?

Seems sensible. Observe that we will need to return the same name (concatenated with `:`) from the property method `iface_name`.